server/handlers/meshery_pattern_handler.go

```diff
@@ -1025,9 +1023,12 @@ func (h *Handler) GetMesheryPatternsHandler(
 	}
 	filter := struct {
 		Visibility []string `json:"visibility"`
+		Populate   []string `json:"populate"`
 	}{}
 
 	visibility := q.Get("visibility")
+	populate := q.Get("populate")
```

Comment:
direrctly do q.Get["populate"] it will return an array , we shouldnt be doing manuall parsing . 
to put multiple params in query do ?populate=x&poulate=y